frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarContainer.tsx

```diff
@@ -23,7 +23,7 @@ import { getUser, getUserIsAdmin } from "metabase/selectors/user";
 import {
   getHasDataAccess,
   getHasOwnDatabase,
-} from "metabase/new_query/selectors";
+} from "metabase/new_dataset/selectors";
```

Comment:
Just stepping by and figured it'd be more helpful to ping you earlier than later. Please use "model" instead of "dataset" for the new code. All the existing code using "dataset" was introduced with a WIP name in mind, and we'd need to spend some time renaming all that eventually; but for new code please use "model"

Reply:
Cool @kulyk, yes, great that we're choosing one and going with it 👍